src/options.js

```diff
@@ -54,11 +54,10 @@ Options.extract = function (document) {
       options.https = element.src.indexOf('https') === 0;
 
       options.host = host;
-      options.port = port
-        ? parseInt(port, 10)
-        : portFromAttr
-          ? parseInt(portFromAttr, 10)
-          : options.port;
+      const ourPort = parseInt(port || portFromAttr) || ''; // port script is running from
```

Comment:
The comment is missing a part of the sentence, right?
Also, please call `parseInt` using the [radix](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#parameters) parameter (`parseInt(port || portFromAttr, 10)`) to avoid confusion.

Reply:
> The comment is missing a part of the sentence, right?

Rather me omitting too much in an attempt to keep the line width short; it should read as "the port that the script is running from". In the rebase I have rewritten this to a separate comment, however.